deploy/helm/kubecf/assets/operations/instance_groups/credhub.yaml

```diff
@@ -60,11 +60,11 @@
               name: {{ printf "%s.var-router-ssl" .Release.Name | quote }}
               key: ca
         - name: CF_USERNAME
-          value: admin
+          value: credhub_setup
         - name: CF_PASSWORD
           valueFrom:
             secretKeyRef:
-              name: {{ printf "%s.var-cf-admin-password" .Release.Name | quote }}
+              name: {{ printf "%s.var-credhub-setup-client-secret" .Release.Name | quote }}
```

Comment:
Looks ugly, but I guess because we don't have a good way to reference the secret using operator things… we can't just `((credhub_setup_client_secret))` because then it isn't treated as a secret anymore :(

Reply:
Yep.